### PR TITLE
feat: extend app config

### PR DESCRIPTION
**Pull request by peterdeme**
- Extend app configs
- Fix failing tests. Apparently there are always 5 commands that stays.

### CODE REVIEW

**Review by driver-devel — DISMISSED** [review]
LGTM

File: src/test/java/io/getstream/chat/java/BasicTest.java [hunk 1]
```diff
@@ -83,7 +83,7 @@ private static void cleanCommands() throws StreamException {
         () -> {
           var commands =
               Assertions.assertDoesNotThrow(() -> Command.list().request().getCommands());
-          return commands == null || commands.size() == 0;
+          return commands.size() == 5; // Built-in 5 commands
```
Comment (ferhatelmas) [inline src/test/java/io/getstream/chat/java/BasicTest.java:86]: correct

File: src/main/java/io/getstream/chat/java/models/App.java [hunk 2]
```diff
@@ -100,6 +100,14 @@
     private String dataTemplate;
   }
 
+  @Data
+  @NoArgsConstructor
+  public static class HuaweiConfig {
```
Comment (ferhatelmas) [inline src/main/java/io/getstream/chat/java/models/App.java:105]: how about configuration ?

Comment (peterdeme) [reply src/main/java/io/getstream/chat/java/models/App.java:105]: I have no clue how I missed it. Added ✅

**Review by ferhatelmas — APPROVED** [review]
LGTM

File: src/main/java/io/getstream/chat/java/models/App.java (hunk 2 above)
Comment (ferhatelmas) [reply src/main/java/io/getstream/chat/java/models/App.java:105]: 👍🏻

**Pull request merged**